src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApiTest.java

```diff
@@ -0,0 +1,172 @@
+        given(authenticatorFactory.createForJurisdiction(anyString())).willReturn(ccdAuthenticator);
+        given(ccdAuthenticator.getServiceToken()).willReturn("serviceToken");
+        doThrow(
+            new FeignException.BadRequest("Msg", "Body".getBytes()))
+            .when(feignCcdApi).startEventForCaseWorker(
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString()
+            );
+
+        // when
+        CcdCallException ccdCallException = catchThrowableOfType(() ->
+            ccdApi.startAttachScannedDocs(
+                caseDetails,
+                "idamToken",
+                "userId"
+            ), CcdCallException.class
+        );
+
+        // then
+        assertThat(ccdCallException.getCause().getMessage()).isEqualTo("Msg");
+    }
+
+    @Test
+    void should_rethrow_feign_exception_in_getCase() {
+        // given
+        given(authenticatorFactory.createForJurisdiction(anyString())).willReturn(ccdAuthenticator);
+        given(ccdAuthenticator.getUserToken()).willReturn("userToken");
+        given(ccdAuthenticator.getServiceToken()).willReturn("serviceToken");
+        doThrow(new FeignException.BadRequest("Msg", "Body".getBytes()))
+            .when(feignCcdApi).getCase(
+            anyString(),
+            anyString(),
+            anyString()
+            );
+
+        // when
+        InvalidCaseIdException invalidCaseIdException = catchThrowableOfType(() ->
+            ccdApi.getCase(
+                "caseRef",
+                "BULKSCAN"
+            ), InvalidCaseIdException.class
+        );
+
+        // then
+        assertThat(invalidCaseIdException.getCause().getMessage()).isEqualTo("Msg");
+    }
+
+    @Test
+    void should_rethrow_feign_exception_in_startEvent() {
+        // given
+        given(ccdAuthenticator.getUserToken()).willReturn("userToken");
+        given(ccdAuthenticator.getServiceToken()).willReturn("serviceToken");
+        given(ccdAuthenticator.getUserDetails()).willReturn(userDetails);
+        given(userDetails.getId()).willReturn("userId");
+        doThrow(new FeignException.BadRequest("Msg", "Body".getBytes()))
+            .when(feignCcdApi).startEventForCaseWorker(
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString(),
+            anyString()
+            );
+
+        // when
+        FeignException feignException = catchThrowableOfType(() ->
+            ccdApi.startEvent(
+                ccdAuthenticator,
+                "jurisdiction",
+                "caseTypeId",
+                "caseRef",
+                "eventTypeId"
+            ), FeignException.class
+        );
+
+        // then
+        assertThat(feignException.getMessage()).isEqualTo("Msg");
+    }
+
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4567585)